app/views/application/_we_share.html.erb

```diff
@@ -1,5 +1,8 @@
 <section id="share">
   <h3 class="share-title is-not-mobile">We share what we know</h3>
+  <div class="categories index categories-mobile">
+    <%= select_tag :category, options_from_collection_for_select(@categories, 'id', 'name'), class: 'categories-select' %>
```

Comment:
@jaime-evr I always try to write lines with 80 characters or less in this case I would prefer to format this line as follow:
```
<%= select_tag :category,
                          options_from_collection_for_select(@categories, 'id', 'name'),
                          class: 'categories-select' %>
```